baywatch.install

```diff
@@ -324,3 +324,33 @@ function baywatch_update_8013() {
   $baywatch = new BaywatchOperation();
   $baywatch->enable_tide_ui_restriction();
 }
+
+/**
+ * Remove Elasticsearch Connector configuration from config_split
+ */
+function baywatch_update_8014() {
+    // config_split files
+    $configs = [
+        'config_split.config_split.ci' => 'config_split',
+        'config_split.config_split.dev' => 'config_split',
+        'config_split.config_split.local' => 'config_split',
```

Comment:
it doesn't need a two-dimensional array here, you can just loop through each of the values. In the code below you are not using the type anywhere. So you can just do it like this - 
```
$configs = [
        'config_split.config_split.ci',
        'config_split.config_split.dev',
        'config_split.config_split.local',
]
foreach ($configs as $config) {
  // then the same code
}
```

Reply:
Done